routes/web.php

```diff
@@ -16,4 +16,6 @@
 
 Route::get('/b/{language}', 'MainController@languageBoard')->name('board.list');
 
-Route::get('/l/{label_name}', 'MainController@labelBoard')->name('label.list');
\ No newline at end of file
+Route::get('/l/{label_name}', 'MainController@labelBoard')->name('label.list');
+
+Route::get('/l', 'MainController@labelsList')->name('labels.list');
```

Comment:
I would change to `labels.all` to make a clear distinction between this route and the one above, possibly apply the same concept to the controller method too?

Reply:
Made the suggested changes.